### DISCUSSION TITLE
What is the suggested API to create multiple subscriptions?

### DISCUSSION THREAD

**Discussion by Anurag870** [swellstores/community#147] [Help] [answered]
What is the suggested API to created multiple subscriptions in swell? 

I could find https://developers.swell.is/backend-api/orders/create-an-order
here we can checkout multiple product in single orderand define type -> standard or subscription

Is this correct API?

**Comment by sags95** [comment, marked as answer]
Hi @Anurag870 , with the Backend API you can certainly use the `/orders` endpoint to create multiple subscriptions. You would need to specify the `purchase_option` details like `type` and `plan_id` for each item. Most would use a cart instead of outright creating an order however, this would look quite similar but you would [convert the cart](https://developers.swell.is/backend-api/carts/convert-carts-to-orders) instead of creating an order.

We also do have a Frontend API that you can use as well if you prefer doing this client-side: https://developers.swell.is/frontend-api/introduction
